src/AzureFunctions.PowerShell.Durable.SDK.psm1

```diff
@@ -22,6 +22,16 @@ function GetDurableClientFromModulePrivateData {
     }
 }
 
+function GetInvocationIdFromModulePrivateData {
+    $PrivateData = $PSCmdlet.MyInvocation.MyCommand.Module.PrivateData
+    if ($null -eq $PrivateData -or $null -eq $PrivateData['InvocationId']) {
+        return $null
+    }
+    else {
+        $PrivateData['InvocationId']
```

Comment:
Tiny non-blocking nit: you use `return` in the first branch but omit it here. It works the same, but this inconsistency makes the reader wonder if the intent was different.